### PR TITLE
Fixes #18294: fix display of AK subscription tables.

### PR DESCRIPTION
**Pull request by waldenraines**
http://projects.theforeman.org/issues/18294

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-subscriptions-list.html [hunk 1]
```diff
@@ -36,8 +36,12 @@
 
       <tbody>
         <tr bst-table-row ng-repeat-start="(name, subscriptions) in groupedSubscriptions | groupedFilter: subscriptionSearch">
-          <td bst-table-cell class="row-select" style="white-space:nowrap">{{ name }}</td>
-          <td bst-table-cell colspan="8" ></td>
+          <td class="row-select"></td>
+          <td colspan="8">
+            <a href='/subscriptions/{{ subscription.id }}/info' class="confined-text">
```
Comment (jlsherrill) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-subscriptions-list.html]: this doesn't work for me, as there is no 'subscription' variable here, only subscriptions. 'subscription' is defined below in the ng-repeat, but not here.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-add-subscriptions.html [hunk 2]
```diff
@@ -37,8 +37,12 @@
 
       <tbody>
         <tr bst-table-row ng-repeat-start="(name, subscriptions) in groupedSubscriptions">
-          <td bst-table-cell class="row-select" style="white-space:nowrap">{{ name }}</td>
-          <td bst-table-cell colspan="8"></td>
+          <td class="row-select"></td>
+          <td colspan="8">
+            <a href='/subscriptions/{{ subscription.id }}/info' class="confined-text">
```
Comment (jlsherrill) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-add-subscriptions.html]: same issue here

**Comment by waldenraines** [comment]
@jlsherrill fixed.

**Comment by jlsherrill** [comment]
Just realized the link right below this one here:  https://github.com/waldenraines/katello/blob/92e7185ca1d65728d4ff42da0fdbfd1f3df66215/engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-add-subscriptions.html#L62

is pointing to an invalid url /info doesn't exist, should just be  /subscriptions/id/.  Do you wanna just fix it here? or want me to file an issue?

**Comment by waldenraines** [comment]
> Just realized the link right below this one here: https://github.com/waldenraines/katello/blob/92e7185ca1d65728d4ff42da0fdbfd1f3df66215/engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-add-subscriptions.html#L62
> 
> is pointing to an invalid url /info doesn't exist, should just be /subscriptions/id/. Do you wanna just fix it here? or want me to file an issue?

Fixed all places where /info was being used.  Also fixed the links on the content host subscriptions pages that these were based on and which were also going no where.

**Pull request merged**